AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/hydra_SubID_NoChange_MeshChanged.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    asset_id_no_change = (
+        "Model AssetId matches after changes",
+        "P0: Model AssetId does not match expected output")
+
+def SubID_NoChange_MeshChanged():
+
+    import os
+    import shutil
+    import time
+    from pathlib import Path
+
+    from Atom.atom_utils.atom_constants import AtomComponentProperties
+    from EditorPythonTestTools.editor_python_test_tools.editor_entity_utils import EditorEntity
+    from EditorPythonTestTools.editor_python_test_tools.utils import Report, TestHelper, Tracer
+
+    with Tracer() as error_tracer:
+        # Test Setup: Wait for Editor idle before loading the level and executing Python hydra scripts.
+        TestHelper.init_idle()
+        TestHelper.open_level("AssetPipeline", "SceneTests")
+
+        # Test Setup: Set up source and destination for assetinfo file, then verify there is no file there already.
+        dirpath = Path(__file__).parents[4]
```

Comment:
what is the 4 here? Can you make it a variable to help with long term maintenance?

Reply:
The 4 is how many directories we are going up the tree to find the AutomatedTesting folder.